src/data-table/column-categorical.js

```diff
@@ -57,43 +59,11 @@ function FilterQuickControls(props: {
   onSelectAll: () => void,
   onClearSelection: () => void,
 }) {
-  const [css, theme] = useStyletron();
   return (
-    <React.Fragment>
-      <button
-        type="button"
-        className={css({
-          ...theme.typography.font100,
-          backgroundColor: theme.colors.backgroundPrimary,
-          borderTop: 0,
-          borderRight: 0,
-          borderBottom: 0,
-          borderLeft: 0,
-          color: theme.colors.contentPrimary,
-          cursor: 'pointer',
-        })}
-        onClick={props.onSelectAll}
-      >
-        Select All
-      </button>
-      <span className={css({...theme.typography.font100})}> | </span>
-      <button
-        type="button"
-        className={css({
-          ...theme.typography.font100,
-          backgroundColor: theme.colors.backgroundPrimary,
-          borderTop: 0,
-          borderRight: 0,
-          borderBottom: 0,
-          borderLeft: 0,
-          color: theme.colors.contentPrimary,
-          cursor: 'pointer',
-        })}
-        onClick={props.onClearSelection}
-      >
-        Clear
-      </button>
-    </React.Fragment>
+    <ButtonGroup size={SIZE.mini} kind={KIND.minimal}>
+      <Button onClick={props.onSelectAll}>Select All</Button>
+      <Button onClick={props.onClearSelection}>Clear</Button>
```

Comment:
These buttons should have `type="button"` I believe. Clicking on either submits the form.